<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>272</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="homeTab">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QProgressBar" name="battProgBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pauseButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pauseButton 
{
	background-color: transparent;
	border-image: url(:/img/pause_icon_96.png);
	background: none;
	border: none;
	background-repeat: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="playButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>200</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#playButton 
{
	background-color: transparent;
	border-image: url(:/img/play_icon_100.png);
	background: none;
	border: none;
	background-repeat: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="stopButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>200</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#stopButton 
{
	background-color: transparent;
	border-image: url(:/img/stop_icon_100.png);
	background: none;
	border: none;
	background-repeat: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="currentSpeedLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>90 km/h</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="speedLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed:</string>
      </property>
     </widget>
     <widget class="QLabel" name="battCapLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Battery capacity remaining:</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="rangeBox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;69 km&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="rangeLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Range:</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="modeBox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Race&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="modeLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Mode:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ddTab">
     <attribute name="title">
      <string>Driving Details</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="batteryTab">
     <attribute name="title">
      <string>Battery</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="mpTab">
     <attribute name="title">
      <string>Music Player</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="alarmTab">
     <attribute name="title">
      <string>Alarms</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
